vip_spec.xsd

```diff
@@ -511,13 +507,13 @@ reference elements for polling location
 
     could be:
 
-      <Office textIdentifier="office_mayor">
+      <Office identifier="office_mayor">
         <text language="en">Mayor</text>
         <text language="es">Alcalde</text>
         <text language="zh">市長</text>
```

Comment:
I know this isn't part of the PR, but could you fix the casing of the `<Text>` tags here?